tutorials/plot_object_raw.py

```diff
@@ -30,15 +31,21 @@
 # Load an example dataset, the preload flag loads the data into memory now
 data_path = op.join(mne.datasets.sample.data_path(), 'MEG',
                     'sample', 'sample_audvis_raw.fif')
-raw = mne.io.RawFIF(data_path, preload=True, verbose=False)
+raw = mne.io.read_raw_fif(data_path, preload=True, verbose=False)
 
 # Give the sample rate
 print('sample rate:', raw.info['sfreq'], 'Hz')
 # Give the size of the data matrix
 print('channels x samples:', raw._data.shape)
 
 ###############################################################################
-# Information about the channels contained in the :class:`Raw <mne.io.RawFIF>`
+# .. note:: Accessing the `._data` attribute is done here for educational
+#           purposes. However this is a private attribute as its name starts
+#           with an `_`. This suggests that you should **not** access this
+#           variable directly but rely on indexing syntax detailed just below.
```

Comment:
Yeah it's fine this way, I just figured it's not necessary for users since you could just say that "The core data structure is simply a 2D numpy array (channels × samples) which can be access through indexing the Raw object"